src/mixins/selection.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+/**
+ * CSS to style the selection psuedo-element.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...selection({
+ *     'background': 'blue'
+ *   }, 'section')
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   ${selection({'background': 'blue'}, 'section')}
+ * `
+ *
+ * // CSS as JS Output
+ *
+ * 'div': {
+ *   'section::-moz-selection': {
+ *     'background-color':'blue',
+ *   },
+ *   'section::selection': {
+ *     'background-color': 'blue',
+ *   }
+ * }
+ */
+
+function selection(styles: Object, parent: string = '') {
```

Comment:
I lean towards no on this one. Unlike other pseudo selectors `::selection` is allowed and often the most common use case, where other pseudos need a parent to function.